sonar-plugin-api/src/main/java/org/sonar/api/web/DashboardTemplate.java

```diff
@@ -26,7 +26,9 @@
  * This extension point must be implemented to define a new dashboard.
  *
  * @since 2.13
+ * @deprecated since 6.2, as dahsboards have been removed
  */
+@Deprecated
 @ServerSide
 @ExtensionPoint
 public abstract class DashboardTemplate {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)